src/scripts.js

```diff
@@ -78,19 +81,27 @@ function displayAllRecipes() {
   } 
 }
 
-function displayPantryView() {
-  pantryView.classList.remove('hidden');
-  allRecipesView.classList.add('hidden');
-  buttonPantry.classList.add('hidden');
-  buttonAllRecipes.classList.remove('hidden');
+function displayPantryView() { //refactor if time!
+  pantryView.classList.toggle('hidden');
+  allRecipesView.classList.toggle('hidden');
+  buttonPantry.classList.toggle('hidden');
+  buttonAllRecipes.classList.toggle('hidden');
+  pantryUserName.innerText = currentUser.name;
+  diplayUserPantryIngredients();
 }
 
-function displayAllRecipesView() {
-  pantryView.classList.add('hidden');
-  allRecipesView.classList.remove('hidden');
-  buttonPantry.classList.remove('hidden');
-  buttonAllRecipes.classList.add('hidden');
+function diplayUserPantryIngredients() {
+  let ingredientNames = currentUser.pantry.getIngredientName(sampleIngredients);
+  console.log("give me my ingredients!!")
+  allPantry.innerHTML = ''
+  for(var i = 0; i < ingredientNames.length; i++) {
```

Comment:
I know it was easy to work with a `for `loop here. We would need to refactor `for` loop to probably forEach.